### PR TITLE
ES6 refactor

### PR DESCRIPTION
**Pull request by gkal19**

### CODE REVIEW

**Comment by zenorocha** [comment]
This is great, thanks @gkal19!

There's only one problem, in external facing code like demos and docs we can't have ES6 code. The reason is because it's not supported on the browsers that clipboard.js support.

For example, clipboard.js supports IE 9+ (https://github.com/zenorocha/clipboard.js#browser-support), but arrow functions (http://caniuse.com/#search=arrow) are not supported on IE11 and below.

**Comment by gkal19** [comment]
@zenorocha, it is a great pleasure to be contributing to your project.
The demo codes have been restored to the original codes.

**Comment by zenorocha** [comment]
Cool, I noticed you were adding commits so let me know when you're done ;)

**Comment by gkal19** [comment]
@zenorocha , This is my last commit. Feel free to review the Pull Request.

File: .travis.yml [hunk 1]
```diff
@@ -2,3 +2,5 @@ sudo: false
 language: node_js
 node_js:
   - stable
+script:
+  - npm test
```
Comment (zenorocha) [inline .travis.yml]: Is this really necessary?

Comment (gkal19) [inline .travis.yml]: It is true! It is not necessary.

**Comment by zenorocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by zenorocha** [comment]
@gkal19 did you test before sending this PR? I ran `npm run build`, then opened the demos and all of them show this error message on the Console: `Uncaught TypeError: Clipboard is not a constructor`. I believe it's related to `export default ...`.

**Comment by gkal19** [comment]
Damn it! It's true I have a local commit here on my computer but I forgot to send it. In a few hours I'll send.

**Comment by gkal19** [comment]
@zenorocha 
I did not make any mistake when running `npm run build`.

**Comment by zenorocha** [comment]
Thanks @gkal19!

**Comment by loanniss** [comment]
你好，请问这个如何在vue使用？

**Comment by minerchow** [comment]
@loanniss  https://blog.csdn.net/guxuehua/article/details/79169190

**Pull request merged**